easybuild/easyconfigs/l/LIBSVM/LIBSVM-3.20-intel-2015b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp'
+
+name = 'LIBSVM'
+version = '3.20'
+
+homepage = 'http://www.csie.ntu.edu.tw/~cjlin/libsvm/'
+description = """LIBSVM is an integrated software for support vector classification, (C-SVC, nu-SVC), regression
+ (epsilon-SVR, nu-SVR) and distribution estimation (one-class SVM). It supports multi-class classification."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = [homepage]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('Qt', '4.8.7')]
+
+buildopts = ' && cd svm-toy/qt && make MOC=$EBROOTQT/bin/moc '
```

Comment:
Should this be in a subshell? e.g. `&& (cd svm-toy/qt && make MOC=...)` then it will automatically cd back to the original position.